public/data/kumaran.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "T.kumaran",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/synackred.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/synackred",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing, Pentesting,and Bugbounty Hunting",
```

Maybe you missed the space here 😅 

```suggestion
        "title": "Started Freelancing, Pentesting, and Bugbounty Hunting",
```

---

yeah space added to it